frontend/pages/admin/backups.vue

```diff
@@ -108,8 +108,8 @@
         </div>
       </section>
     </section>
-    <v-container class="mt-4 d-flex justify-end">
-      <v-btn outlined rounded to="/group/migrations"> {{ $t('recipe.looking-for-migrations') }} </v-btn>
+    <v-container class="mt-4 d-flex justify-center text-center">
+      <a to="/group/migrations"> {{ $t('recipe.looking-for-migrations') }} </a>
```

Comment:
Why did you change this to an `a` tag? I believe it should be a Nuxt Link https://nuxtjs.ir/api/components-nuxt-link